Project.toml

```diff
@@ -4,6 +4,14 @@ authors = ["Climate Modeling Alliance and contributors"]
 version = "0.3.0"
 
 [deps]
+Elliptic = "b305315f-e792-5b7a-8f41-49f472929428"
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+ProgressBars = "49802e3a-d2f1-5c88-81d8-b72133a6f568"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
```

Comment:
@siddharthabishnu we worked to minimize the dependencies in https://github.com/CliMA/CubedSphere.jl/pull/38; why you choose to bring them back?

Reply:
I mean I'm asking because mostly I don't see where the PR uses most of it. I'm wondering if it's something related with  how you merge main to this branch?

Reply:
@navidcy I am using 
```
using LinearAlgebra
using Statistics: mean, norm
using Random
```
in `src/non_uniform_conformal_cubed_sphere.jl` and I need the remaining dependencies to run `examples/conformal_cubed_sphere.jl`. Before your comment regarding PR #38, I didn't notice two Project.toml files---one of them being in the sandbox directory. I suspect there's a better alternative where I don't need to add all the dependencies in the `Project.toml` file within the `src` directory.

Reply:
For the example, you should literate it and include it in the docs?

Something like: https://clima.github.io/CubedSphere.jl/stable/conformal_cubed_sphere/#Conformal-cubed-sphere-mapping

The docs have a different toml file that includes extra dependencies (eg plotting packages and whatever else you might want).

So I suggest we keep the dependencies of the main package to the minimum because other packages depend on CubedSphere and it's good to keep it lightweight (it's good to keep the dependencies to the minimum anyway!).
The imports you point above that you need all part of Julia base so it's totally fine!

Reply:
Got it. Thanks.

Reply:
@navidcy, I have minimized the dependencies in commit [149c5cd](https://github.com/CliMA/CubedSphere.jl/pull/41/commits/149c5cd15f47db6d4ef94e369cd8eade493c10ef).